app/src/lib/app-state.ts

```diff
@@ -222,6 +222,7 @@ export type Popup =
       type: PopupType.ConfirmDiscardChanges
       repository: Repository
       files: ReadonlyArray<WorkingDirectoryFileChange>
+      forceConfirmation: boolean
```

Comment:
We've not really been good at writing docs for our popup types, but would you mind adding some docs to this field to help the caller understand what it represents?